gui/packages/ubuntupro/lib/pages/subscription_status/subscription_status_page.dart

```diff
@@ -25,19 +26,19 @@ class SubscriptionStatusPage extends StatelessWidget {
       child: switch (model) {
         StoreSubscriptionStatusModel() => SubscriptionStatus(
             caption: lang.storeManaged,
-            actionButton: TextButton(
+            actionButton: OutlinedButton(
               onPressed: model.launchManagementWebPage,
-              style: TextButton.styleFrom(
-                foregroundColor: const Color(0xFF0094FF),
-              ),
               child: Text(lang.manageSubscription),
             ),
           ),
         UserSubscriptionStatusModel() => SubscriptionStatus(
-            caption: lang.manuallyManaged('ubuntu.com/pro/dashboard'),
+            caption: lang.manuallyManaged(
+              'ubuntu.com/pro/dashboard',
+              'https://ubuntu.com/pro/dashboard',
+            ),
             actionButton: FilledButton(
               style: FilledButton.styleFrom(
-                backgroundColor: const Color(0xFFE86581),
+                backgroundColor: YaruColors.red,
```

Comment:
There is a todo item above you could have deleted, since you did what it was about. BTW I forgot to ask during the hangout: did you try `YaruColors.error` instead of `YaruColors.red`? Having some semantic names for the colors is a nice thing.

Reply:
There is `YaruColors.dark.error` and `YaruColors.light.error` but they didn't look as good or as close to the design as `YaruColors.red`.